packages/backend/src/videos/videos.controller.ts

```diff
@@ -19,15 +19,9 @@ export class VideosController{
         return this.videosService.getAll(param);
     }
 
-    @Get('category')
-    getAllCategoryList(){
-        return this.videosService.getAllCategoryList();
-    }
-
-    @ApiParam({ name: 'id', type:'string' })
-    @Get('/:params')
-    getStream(@Query() id: string ,@Res() res, @Req() req) {
-        this.videosService.getStream(id, res, req);
+    @Get(['category/:only','category'])
+    getAllCategoryList(@Param('only') only:string){
+        return this.videosService.getAllCategoryList(only === "only");

```

Comment:
Zrobił bym to bardziej jak justyna w showAllVideos. Jesli przyjmowany jest parametr np. OnlyCategory, WithVideo to powinien byc to enum

Reply:
poprawione